### ISSUE TITLE
suggestion for improvement : add the .ssh folder located in org.xbmc.kodi/files/ to the backup

### ISSUE THREAD

**Issue by MehdiTouns31** [robweber/xbmcbackup#242]
suggestion for improvement: 
is it possible to add the .ssh folder located in org.xbmc.kodi/files/ to the backup?
This folder contains the essential ssh key pair for accessing SFTP server sources. This would be super useful especially on Smartphones running Android 13+ where you cannot access this folder using file explorer... :(

Thanks for all
See you

**Comment by robweber** [comment]
Just to get more clarity is this folder only on Kodi when running Android?

**Comment by MehdiTouns31** [comment]
No, I think the .ssh folder is created by default on all Kodi platforms and contains files configurations as soon as a media source using the SFTP protocol has been defined.
So it’s possible that, in some cases, the folder may be empty.

**Comment by MehdiTouns31** [comment]
If you need any other information, I am available to provide it to you. 

Thank you for your work. The backup addon is so practical.
